src/codecoverage/backend/requirements_frozen.txt

```diff
@@ -48,14 +51,16 @@ flask-talisman==0.6.0
 flit==1.3
 future==0.17.1
 google-api-core==1.10.0
+google-api-python-client==1.7.7
 googleapis-common-protos==1.5.10
-google-auth==1.6.3
+google-auth==1.6.2
```

Comment:
This was downgraded, expected?

Reply:
Useless now since i removed ccov backend changes